giraffe/src/types/index.ts

```diff
@@ -97,10 +97,12 @@ export interface Table {
   getColumn: GetColumn
   getColumnName: (columnKey: string) => string | null // null if the column is not available
   getColumnType: (columnKey: string) => ColumnType | null // null if the column is not available
+  getOriginalColumnType: (columnKey: string) => FluxDataType | null // null if the column is not available
   columnKeys: string[]
   length: number
   addColumn: (
     columnKey: string,
+    originalType: FluxDataType,
```

Comment:
Can we use `fluxDataType` as the property name here? `originalType` does not sound meaningful to me. There is a quick way to replace all instances of that property name in all files using VS Code.

Reply:
Definitely. Great idea